lte/gateway/c/core/oai/lib/n11/M5GAuthenticationServiceClient.h

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <grpc++/grpc++.h>
+#include "lte/protos/subscriberauth.grpc.pb.h"
+#include "lte/protos/subscriberauth.pb.h"
+#include "includes/GRPCReceiver.h"
+#include <stdint.h>
+#include <functional>
+#include <memory>
+#include "3gpp_38.413.h"
+
+using grpc::Status;
+using magma::GRPCReceiver;
+using magma::lte::M5GAuthenticationInformationAnswer;
+using magma::lte::M5GAuthenticationInformationRequest;
+using magma::lte::M5GSubscriberAuthentication;
+
+namespace magma5g {
+
+class M5GAuthenticationServiceClient {
```

Comment:
add test coverage

Reply:
Comment addressed.

Reply:
partially. you need to create mock handlers for both `get_subs_auth_info` and `get_subs_auth_info_resync`. As is you are simply testing a function that is called within `get_subs_auth_info`.

Reply:
The Unit Test will be covered in next PR.